artemis-ra/src/main/java/org/apache/activemq/artemis/ra/referenceable/ActiveMQRAConnectionFactoryObjectFactory.java

```diff
@@ -27,7 +27,7 @@
  *
  * Given a reference - reconstructs an ActiveMQRAConnectionFactory
  */
-public class ConnectionFactoryObjectFactory implements ObjectFactory {
+public class ActiveMQRAConnectionFactoryObjectFactory implements ObjectFactory {
```

Comment:
is that really needed? Thinking of Wildfly again here.

Perhaps an extension?

Reply:
was a tidy up, again not needed but just being explicit. we can rename and extend.